env0/provider.go

```diff
@@ -42,11 +43,12 @@ func Provider() *schema.Provider {
 			"env0_aws_credentials":        dataAwsCredentials(),
 		},
 		ResourcesMap: map[string]*schema.Resource{
-			"env0_project":                resourceProject(),
-			"env0_configuration_variable": resourceConfigurationVariable(),
-			"env0_template":               resourceTemplate(),
-			"env0_ssh_key":                resourceSshKey(),
-			"env0_aws_credentials":        resourceAwsCredentials(),
+			"env0_project":                		resourceProject(),
+			"env0_configuration_variable": 		resourceConfigurationVariable(),
+			"env0_template":               		resourceTemplate(),
+			"env0_ssh_key":                		resourceSshKey(),
+			"env0_aws_credentials":        		resourceAwsCredentials(),
+			"env0_template_project_assignment": resourceTemplateProjectAssignment(),
```

Comment:
formatting

Reply:
this is very weird, in VS code it looks fine.
checking

Reply:
![image](https://user-images.githubusercontent.com/76279454/120200929-8bd15200-c22d-11eb-934c-dd145136b3e3.png)

Reply:
3eee432ffaef6f34dd6b16a7ea848fa1c5699298

Reply:
not critical